backend/Controllers/SightingController.cs

```diff
@@ -3,19 +3,29 @@
 using WhaleSpotting.Models.Response;
 using WhaleSpotting.Models.Database;
 using WhaleSpotting.Models.Request;
+using WhaleSpotting.Helpers;
+using System;
 
-namespace WhaleSpotting.Controllers {
+namespace WhaleSpotting.Controllers
+{
     [ApiController]
     [Route("/sightings")]
-    public class SightingController : ControllerBase {
+    public class SightingController : ControllerBase
+    {
         private readonly ISightingService _sightings;
+        private readonly IAuthService _authService;
```

Comment:
You put them in alphabetical order below, but not here! For shame 😉 
```suggestion
        private readonly IAuthService _authService;
        private readonly ISightingService _sightings;
```